bigquery_etl/cli/monitoring.py

```diff
@@ -144,6 +156,164 @@ def deploy(
+        templates_dir="",
+        **jinja_params,
+    )
+
+    statements = []
+    for statement in sqlglot.parse(rendered_result, read="bigquery"):
+        if statement is None:
+            continue
+
+        for select_statement in statement.find_all(sqlglot.exp.Select):
+            statements.append(select_statement)
+
+    return statements
+
+
+@monitoring.command(
+    help="""
+    Deploy custom SQL rules.
+    """
+)
+@click.argument("name")
+@project_id_option()
+@sql_dir_option
+@click.option(
+    "--workspace",
+    default=463,
+    help="Bigeye workspace to use when authenticating to API.",
+)
+@click.option(
+    "--base-url",
+    "--base_url",
+    default="https://app.bigeye.com",
+    help="Bigeye base URL.",
+)
+def deploy_custom_rules(
+    name: str,
+    sql_dir: Optional[str],
+    project_id: Optional[str],
+    base_url: str,
+    workspace: int,
+) -> None:
+    """Deploy custom SQL rules for files."""
+    api_key = os.environ.get("BIGEYE_API_KEY")
+    if api_key is None:
+        click.echo(
+            "Bigeye API token needs to be set via `BIGEYE_API_KEY` env variable."
+        )
+        sys.exit(1)
+
+    custom_rules_files = paths_matching_name_pattern(
+        name, sql_dir, project_id=project_id, files=[CUSTOM_RULES_FILE]
+    )
+
+    api_auth = APIKeyAuth(base_url=base_url, api_key=api_key)
+    client = datawatch_client_factory(api_auth, workspace_id=workspace)
+    collections = client.get_collections()
+    warehouse_id = ConfigLoader.get("monitoring", "bigeye_warehouse_id")
+    existing_rules = client.get_rules_for_source(warehouse_id=warehouse_id)
+    existing_rules_sql = [rule.custom_rule.sql for rule in existing_rules.custom_rules]
+    existing_schedules = {
+        schedule.name: schedule.id
+        for schedule in client.get_named_schedule().named_schedules
+    }
+    url = "/api/v1/custom-rules"
+
+    for custom_rule_file in list(set(custom_rules_files)):
+        project, dataset, table = extract_from_query_path(custom_rule_file)
+        try:
+            metadata = Metadata.from_file(custom_rule_file.parent / METADATA_FILE)
+            if metadata.monitoring and metadata.monitoring.enabled:
+                # Convert all the Airflow params to jinja usable dict.
+                for select_statement in _sql_rules_from_file(
+                    custom_rule_file, project, dataset, table
+                ):
+                    sql = select_statement.sql(dialect="bigquery")
+                    if sql in existing_rules_sql:
+                        continue
+
+                    # parse config values from SQL comment
+                    config = {}
+                    if select_statement.comments:
+                        config = ast.literal_eval("".join(select_statement.comments))
+
+                    payload = {
+                        "name": config.get(
+                            "name", f"{project}_{dataset}_{table}_bqetl_check"
```

Comment:
```suggestion
                            "name", f"{project}_{dataset}_{table}_bigeye_check"
```

Maybe just to avoid confusion around the exiting bqetl checks and bigeye monitoring?